phpcs.xml.dist

```diff
@@ -12,7 +12,7 @@
 
 	<!-- Configs -->
 	<config name="minimum_supported_wp_version" value="4.7" />
-	<config name="testVersion" value="5.2-"/>
+	<config name="testVersion" value="7.0-"/>
```

Comment:
Since this code runs on Atomic, which iirc is 7.2, this seems like a safe change to make.